testdata/config/metrics.yaml

```diff
@@ -6,11 +6,10 @@ metadata:
 spec:
   value: "1"
   dimensions:
-    source: source.labels["app"] | "unknown"
-    target: target.service | "unknown"
-    service: target.labels["app"] | "unknown"
-    method: request.path | "unknown"
-    version: target.labels["version"] | "unknown"
+    source_service: source.service | "unknown"
+    source_version: source.labels["version"] | source.labels["istio"] | "unknown"
```

Comment:
Why do we use the "istio" label for version?

Reply:
this a good catch. this slipped in when I was bouncing ideas off of people -- the dangers of not checking your current branch.

Reply:
Fixed.